compiler/rustc_smir/src/rustc_smir/convert/mir.rs

```diff
@@ -151,6 +151,12 @@ impl<'tcx> Stable<'tcx> for mir::StatementKind<'tcx> {
             mir::StatementKind::ConstEvalCounter => {
                 stable_mir::mir::StatementKind::ConstEvalCounter
             }
+            mir::StatementKind::BackwardIncompatibleDropHint { place, reason } => {
+                stable_mir::mir::StatementKind::BackwardIncompatibleDropHint {
+                    place: place.stable(tables),
+                    reason: reason.stable(tables),
+                }
+            }
```

Comment:
Here we can translate to `Nop` instead. Note that this will require format change.

```suggestion
            mir::StatementKind::BackwardIncompatibleDropHint { .. } |
```

Reply:
Or I guess this would be equivalent:
```suggestion
            mir::StatementKind::BackwardIncompatibleDropHint { .. } => stable_mir::mir::StatementKind::Nop,
```